packages/front-end/components/Metrics/DateGraph.tsx

```diff
@@ -9,8 +10,21 @@ import { scaleLinear, scaleTime } from "@visx/scale";
 import { AxisBottom, AxisLeft } from "@visx/axis";
 import { AreaClosed, LinePath } from "@visx/shape";
 import { curveMonotoneX } from "@visx/curve";
+import {
+  TooltipWithBounds,
+  useTooltip,
+  useTooltipInPortal,
+} from "@visx/tooltip";
 import setDay from "date-fns/setDay";
 
+type TooltipData = { x: number; y: number; d: Datapoint };
+interface Datapoint {
+  d: Date | number;
```

Comment:
Wasn't sure the best way to address this. Sometimes it's a date-like numerical key.

Reply:
I think this is fine for now.  Longer term fix is updating all uses of `<DateGraph>` to only pass in Date objects.